.gitignore

```diff
@@ -6,3 +6,4 @@ yarn-error.log*
 
 dist
 superface
+workdir/
```

Comment:
question: I tried to build, run, test the app and the directory was not created. Is it docker related?

Reply:
It is directory we use in onesdk and now in oneservice for adhoc code. Like the one in description of the PR.